examples/manually-instrumented-chatbot/chat-service/export_schema.py

```diff
@@ -0,0 +1,21 @@
+"""
+This file generates the OpenAPI schema of our FastAPI app and saves it to file.
+"""
+
+import json
+
+from fastapi.openapi.utils import get_openapi
+
+from chat.app import app
+
+with open("../frontend/schema.json", "w") as f:
+    json.dump(
+        get_openapi(
+            title="Chat Service Schema",
+            version="1.0.0",
+            description="Your API Description",
```

Comment:
Should fill this out?

Reply:
addressed in 8752d4f8aed347787e7b6d510995c3427ac3bd0e